include/questdb/ingress/line_sender.hpp

```diff
@@ -95,6 +99,23 @@ namespace questdb::ingress
         https,
     };
 
+    struct line_sender_buffer_bytes {
```

Comment:
Move this to `#else` branch of `#if __cplusplus >= 202002L`.
No point defining it for C++20 if it's unused.

Also consider inheriting from the C type and adding the accessor methods for ptr and type that mimick the `std::span<const std::byte>` API.